### PR TITLE
Add tests for TFTP read and write requests

### PR DESCRIPTION
**Pull request by shivan1b**
Closes redmine ticket #3043.

### CODE REVIEW

**Comment by catenacyber** [comment]
Looks good to me

File: tests/tftp-tx-handling-rrq/suricata.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
```
Comment (victorjulien) [inline tests/tftp-tx-handling-rrq/suricata.yaml]: you should be able to leave this out as the default config has everything enabled already AFAICS

File: tests/tftp-tx-handling-wrq/suricata.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
```
Comment (victorjulien) [inline tests/tftp-tx-handling-wrq/suricata.yaml]: same here

**Comment by shivan1b** [comment]
Replaced by #88

**Pull request closed without merging**